src/components/hand-controls-child-visibility.js

```diff
@@ -0,0 +1,21 @@
+AFRAME.registerComponent("hand-controls-child-visibility", {
```

Comment:
Since this is now controlling the visibility of the element itself (not the children specifically) it should probably be called `hand-controls-visibility`